src/scripts/main.js

```diff
@@ -1,3 +1,9 @@
 'use strict';
 
-// write code here
+const spider = document.querySelector('.spider');
+const wall = document.querySelector('.wall');
+const difference
+= (wall.clientHeight - spider.clientHeight) / spider.clientHeight;
+
+spider.style.top = `${50 - difference}%`;
+spider.style.left = `${50 - difference}%`;
```

Comment:
But what if the width is different from the height? The spider will not be centred
![image](https://user-images.githubusercontent.com/96409563/228173989-829ba0d2-5cc9-4140-ba34-2f30ff4fd71c.png)